config/routes.rb

```diff
@@ -11,6 +11,16 @@
       to: 'v0/sessions#new',
       constraints: ->(request) { V0::SessionsController::REDIRECT_URLS.include?(request.path_parameters[:type]) }
 
+  match '/vbeta/*path', to: 'application#cors_preflight', via: [:options]
```

Comment:
@patrickvinograd do you know we we had this in the v0 controller? Facilities doesn't modify headers when making requests since it is non-auth, so there shouldn't ever be a need for the browser to do a preflight check.

Reply:
I don't know _why_, but I can confirm that the current facilities controller is receiving OPTIONS requests at a rate roughly equivalent to the rate at which it receives GETs. I agree I don't see why it's needed since it's a pretty simple GET request, no auth or other headers. May be worth asking in the #vetsgov-fe channel to see if we can avoid this?

Reply:
Confirmed it's because of the `X-Key-Inflection` header that we pass to trigger olive branch to translate between snake_case and camelCase.